docs/bis/ec/kindofquantity.md

```diff
@@ -59,8 +59,11 @@ The below represents the string literal syntax. All italicized values are to be
   - `formatName(precision)[First Unit][Second Unit][Third Unit][Fourth Unit]`
 - Overrides may not change units if any are originally defined in the base format, they may only change the display label.
 - Up to 4 per format string
+- As unit labels are optional, they can be kept unset or they can be set to an empty string.
+- When a unit label is not specified in the override, the label will get defaulted to the display label of the unit. The label if it exists in the composite format will be ignored in this case. See example [Null unit label in override](#format-overrides-with-nullempty-unit-labels).
```

Comment:
@ColinKerr : I believe it will fall back to the label specified by the underlying format if it exists ... test to confirm this first

This test has been covered in "ECDbTestFixture.NullOrEmptyStringUnitLabelsInCompositeFormats" in native and "(a)sync - null or empty unit label override" in itwinjs-core.

If no label is given, the final label defaults to the unit's display label.
The label in the composite format gets ignored the minute you add an override in a KoQ.